build -tests fail

when I executed build-tests after executed build command, I got below error:
```
"C:\Users\peter.chen\Documents\Code\Self\corefx\src\System.Diagnostics.PerformanceCounter\tests\System.Diagnostics.Perf
ormanceCounter.Tests.csproj" (BuildAndTest 目標) (1) ->
(CoreCompile 目標) ->
  CounterCreationDataTests.cs(8,6): error CS1001: Identifier expected [C:\Users\peter.chen\Documents\Code\Self\corefx\s
rc\System.Diagnostics.PerformanceCounter\tests\System.Diagnostics.PerformanceCounter.Tests.csproj]
  CounterCreationDataTests.cs(8,6): error CS1002: ; expected [C:\Users\peter.chen\Documents\Code\Self\corefx\src\System
.Diagnostics.PerformanceCounter\tests\System.Diagnostics.PerformanceCounter.Tests.csproj]
  CounterCreationDataTests.cs(8,14): error CS0116: A namespace cannot directly contain members such as fields or method
s [C:\Users\peter.chen\Documents\Code\Self\corefx\src\System.Diagnostics.PerformanceCounter\tests\System.Diagnostics.Pe
rformanceCounter.Tests.csproj]
  CounterCreationDataTests.cs(8,6): error CS1056: Unexpected character '?' [C:\Users\peter.chen\Documents\Code\Self\cor
efx\src\System.Diagnostics.PerformanceCounter\tests\System.Diagnostics.PerformanceCounter.Tests.csproj]
  CounterCreationDataTests.cs(8,28): error CS1022: Type or namespace definition, or end-of-file expected [C:\Users\pete
r.chen\Documents\Code\Self\corefx\src\System.Diagnostics.PerformanceCounter\tests\System.Diagnostics.PerformanceCounter
.Tests.csproj]
```

Then, I opened `CounterCreationDataTests.cs` in notepad, I found a garbled character at line 8
```
using�System.ComponentModel;
```

After I removed it and entered a white space, It works fine now.

Is this a problem?

Thanks.

--- a/src/System.Diagnostics.PerformanceCounter/tests/CounterCreationDataTests.cs
+++ b/src/System.Diagnostics.PerformanceCounter/tests/CounterCreationDataTests.cs
@@ -5,7 +5,7 @@
 using System;
 using System.Collections;
 using System.Collections.Specialized;
-using System.ComponentModel;
+using System.ComponentModel;
 using Xunit;
 
 namespace System.Diagnostics.Tests

